src/types.ts

```diff
@@ -11,3 +11,20 @@ export type TokensValueLocked = {
 export type tvlsObject<T> = {
   [chain: string]: T;
 };
+
+const chains = [
```

Comment:
There's more chains than the ones listed here and I expect this list to keep on growing so I think it may be best to forfeit some type safety in exchange for flexibility and not having to keep updating it all the time.

![image](https://user-images.githubusercontent.com/80541789/123457126-82909680-d5db-11eb-814e-ecfd86181ddc.png)